### DISCUSSION TITLE
iran-internet-types-part1/

### DISCUSSION THREAD

**Discussion by giscus** [amir-pourmand/hugo-papermod-farsi-template#38] [Announcements]
# iran-internet-types-part1/

شاید قدیم، اینترنت یه چیز لوکس تلقی میشد و به درد کار مهمی هم نمی‌خورد ولی الان کمتر کسی اهمیتش رو منکر می‌شه. از طرفی دیدم که خیلی‌ها به علت ناآشنایی با سرویس‌ها و خدماتی که می‌تونند دریافت کنند، اینترنت مناسبی ندارند و اذیت میشن. این بود که گفتم تجربه خودم رو تو این زمینه به اشتراک بگذارم.
در نظر بگیرید که حرف‌هام زیاد تخصصی نیست و نظرات و تجربیات شخصی منه.

https://aprd.ir/iran-internet-types-part1/

**Comment by Davaji** [comment]
سلام وقت بخیر. ممنون میشم اگه وقت کردید ادامه‌ی این مطلب رو بنویسید که خیلی بهش نیاز دارم.

**Reply by pourmand1376** [reply]
سلام. وقت شما بخیر. 

با تأخیر بسیار زیاد نوشتم ولی امیدوارم به دردتون بخوره. 

https://aprd.ir/iran-internet-types-part2
